libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -791,7 +798,7 @@ private synchronized void postPushUnregister(BroadcastReceiver pushReceiver,
             try {
                 context.unregisterReceiver(pushReceiver);
             } catch (Exception e) {
-            	Log.e("SalesforceSDKManager", "Exception occurred while un-registering.", e);
+                logger.log(SalesforceLogger.Level.ERROR, TAG, "Exception occurred while unregistering", e);
```

Comment:
Could you 
```java
import static SalesforceLogger.Level.*;
```
So you could just write
```java
logger.log(ERROR, TAG, ...)
```

Or should we introduce warn / info / error / debug methods on logger?